pkgs/development/python-modules/biliass/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, protobuf
+}:
+
+buildPythonPackage rec {
```

Comment:
Should add `pythonImportsCheck` here too

Reply:
Done.